components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -339,8 +364,14 @@ func main() {
 			disabled: cfg.Avs.Disabled,
 		},
 		{
-			weight: 2,
-			step:   provisioning.NewLmsActivationStep(cfg.LMS, provisioning.NewProvideLmsTenantStep(lmsTenantManager, db.Operations(), cfg.LMS.Region, cfg.LMS.Mandatory)),
+			weight:   2,
+			step:     provisioning.NewClsProvisionStep(clsConfig, clsProvisioner, db.Operations()),
+			disabled: cfg.Cls.Disabled,
+		},
+		{
+			weight:   2,
+			step:     provisioning.NewLmsActivationStep(cfg.LMS, provisioning.NewProvideLmsTenantStep(lmsTenantManager, db.Operations(), cfg.LMS.Region, cfg.LMS.Mandatory)),
+			disabled: !cfg.Cls.Disabled,
```

Comment:
How can I configure SKR to not use LMS and not use CLS ? there is such requirement

Reply:
Hmm, do we have such a requirement? I am not aware of it. What would you suggest in this case? Introducing a disabled flag for LMS?

Reply:
sometimes LMS is not used (disabled), for example on stage and enabled only for given (configured) GA. Merging this PR would enable LMS for all stage SKRs which is not acceptable.

Reply:
there is already a flag for LMS

Reply:
the NewLmsActivationStep step uses it

Reply:
oh, I missed your changes - it is ok as it is. the NewLmsActivationStep step does the work. no problem